### PR TITLE
Move crio and kubernetes jobs to prow

### LINKED ISSUE #1165: Move kubernetes and crio jobs to run as prow periodics
The jobs updated in https://github.com/openshift/aos-cd-jobs/pull/1162 need to be managed by prow with the prospect of moving them to run by the ci-operator.

@stevekuznetsov @paradigm

### PR DESCRIPTION
**Pull request by bbguimaraes**
Fixes #1165.

@kargakis @stevekuznetsov

### CODE REVIEW

File: sjb/config/test_cases/test_pull_request_crio_ami_rhel.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 parent: 'common/test_cases/crio.yml'
 overrides:
-  timer: 'H H * * *'
```
Comment (bbguimaraes) [inline sjb/config/test_cases/test_pull_request_crio_ami_rhel.yml]: This job is already a postsubmit:

https://github.com/openshift/release/blob/414148d7e853423b110122ddf9ff77aeee0ff265/cluster/ci/config/prow/config.yaml#L583-L590

I'm assuming it does not need to be periodic too.

Comment (kargakis) [reply sjb/config/test_cases/test_pull_request_crio_ami_rhel.yml]: Presubmit but I think it's fine.

**Comment by kargakis** [comment]
You need to run `sjb/generate.sh`

**Comment by bbguimaraes** [comment]
@kargakis: too many things at the same time.

Yes, presubmit.

Yes, generated.

**Comment by stevekuznetsov** [comment]
Pushed changes up

**Comment by ingvagabund** [comment]
So where does the job lives/is triggered now? The last e2e test reported on test-grid is March 13. So the kubernetes CI is not currently run.

[1] https://k8s-testgrid.appspot.com/sig-node-kubelet#kubelet-conformance-aws-e2e-rhel

**Comment by bbguimaraes** [comment]
@ingvagabund: after this PR is merged, the jobs should run in prow every 12h:

https://github.com/openshift/release/pull/682

**Pull request merged**